service-onboarding-build-pack/Jenkinsfile

```diff
@@ -315,6 +347,35 @@ node  {
 			sh "ls -lart"
 
             withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: var_credentialsId, passwordVariable: 'PWD', usernameVariable: 'UNAME']]) {
+				if(scm == "gitlab"){
```

Comment:
Move line 350 to 378 to a function

Reply:
The number of parameters that needs to be send to the method is 10+ , so I think it would be better stay here itself rather than make it a method.